pages/en/cookbook/grafting.mdx

```diff
@@ -0,0 +1,219 @@
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.6
+      language: wasm/assemblyscript
+      entities:
+        - Withdrawal
+      abis:
+        - name: Lock
+          file: ./abis/Lock.json
+      eventHandlers:
+        - event: Withdrawal(uint256,uint256)
+          handler: handleWithdrawal
+      file: ./src/lock.ts
+```
+
+- The Locks' data source is the abi and contract address we will get when we compile and deploy the contract
+- The network should correspond to a indexed network being queried. Since we're running on Goerli testnet, the network is `goerli`
+- The `mapping` section defines the triggers of interest and the functions that should be run in response to those triggers. In this case, we are listening for the `Withdrawal` event and calling the `handleWithdrawal` function when it is emitted.
+
+## Grafting Manifest Definition
+
+Grafting requires adding two new items to the original subgraph manifest:
+
+```yaml
+---
+features:
+  - grafting # feature name
+graft:
+  base: Qm... # subgraph ID of base subgraph
+  block: 1502122 # block number
+```
+
+- `features:` is a list of all used [feature names](developing/creating-a-subgraph/#experimental-features).
+- `graft:` is a map of the `base` subgraph and the block to graft on to. The `block` is the block number to start indexing from. The Graph will copy the data of the base subgraph up to and including the given block and then continue indexing the new subgraph from that block on.
+
+The `base` and `block` values can be found by deploying two subgraphs: one for the base indexing and one with grafting
+
+## Deploying the Base Subgraph
+
+1. Go to the [Graph Studio UI](https://thegraph.com/studio/) and create a subgraph on Goerli testnet called `graft-example`
+2. Follow the directions in the `AUTH & DEPLOY` section on your subgraph page in the `graft-example` folder from the repo
+3. Once finished, verify the subgraph is indexing properly. If you run the following command in the graph playground
+
+```graphql
+{
+  withdrawals(first: 5) {
+    id
+    amount
+    when
+  }
+}
+```
+
+It should return the following:
+
+```
+{
+  "data": {
+    "withdrawals": [
+      {
+        "id": "0x13098b538a61837e9f29b32fb40527bbbe63c9120c250242b02b69bb42c287e5-5",
+        "amount": "0",
+        "when": "1664367528"
+      },
+      {
+        "id": "0x800c92fcc0edbd26f74e19ad058c62008a47c7789f2064023b987028343dd498-3",
+        "amount": "0",
+        "when": "1664367648"
+      }
+    ]
+  }
+}
+```
+
+Once you have verified the subgraph is indexing properly, you can deploy the subgraph with grafting.
+
+## Deploying the Grafting Subgraph
+
+1. Go to the [Graph Studio UI](https://thegraph.com/studio/) and create a subgraph on Goerli testnet called `graft-replacement`
+2. Add your `base` subgraph ID to the `graft` section in the `subgraph.yaml` file. This is the `Deployment ID` of your original `graph-example` subgraph. You can find this in the Graph Studio UI.
```

Comment:
Here I think we are missing the important points: How does the graft replacement subgraph.yaml differ from the original one?